helpers/error_handler/error_message.go

```diff
@@ -0,0 +1,40 @@
+package helpers
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/jackc/pgconn"
+)
+
+// interface from github.com/go-playground/validator/v10
+type FieldError interface {
+	Tag() string
+	Field() string
+	Param() string
+}
+
+func ValidationErrorMessage(err FieldError) error {
+	switch err.Tag() {
+	case "email":
+		return errors.New("Invalid email format")
```

Comment:
Just curious, why changing to `errors.New`? What are the advantages? Seems better but I want to understand as well 😁

Reply:
As I took a look at another packages. Non of those packages return string as an error as Go provides `errors` package for us. Also most of the `golang error handling best practice` suggest to return type `error` if the function has an error.

so should be something like this

```go
func ThisFunctionReturnError() error {
   return errors.New("custom error")
}

func SomeFunc() {
  err := ThisFunctionReturnError()
  
  // To use error string (errors package provide us an interface which include func Error() string {...} function 
  log.Print(err.Error())
}
```

Reply:
Also fix the test in 5093112c6388378bea2a6871005e5a6184e55651

Reply:
Indeed, using `errors.New` should be the preferred way. Thank you for the detailed explanation as well.